src/renderer/components/ui/Button.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import classNames from 'classnames'
+
+type ButtonProps = {
+  children: any
+  onClick?: any
+  className?: any
+  style?: Record<string, string | number>
+  type?: 'primary' | 'danger' | 'secondary'
+  round?: boolean
+  id?: string
+  'aria-label'?: string
+  disabled?: boolean
+}
+
+export default function Button({
+  disabled,
+  id,
+  round,
+  onClick,
+  children,
+  className,
+  style,
+  type,
+}: ButtonProps) {
+  return (
+    <button
+      id={id}
+      className={classNames(
+        'delta-button' + (round ? '-round' : ''),
+        'bold',
+        className,
+        type
+      )}
```

Comment:
If possible we should try to use CSS modules from now on for every newly introduced or refactored styling. We can do this here after rebasing this branch to `master`